lines_in_file.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+lines_in_file()
+{
+ cat $1 | wc -l
+}
+
+if [ $# -eq 0 ]
+then
+ echo "no argument is passed"
+ exit 1
+elif ! [ -f $1 ] | [ -e $1 ] 
```

Comment:
When I pass a directory as an argument, It prints "cat : Is a directory". Your script must check and handle the case when a directory is passed.